### ISSUE TITLE
device in picker dosen't appear quickly it take alot of time to appear 

### ISSUE THREAD

**Issue by salah-mohammed** [ConnectSDK/Connect-SDK-iOS#250]
SSDPSocketListener handler take alot of time
when try to add com.apple.developer.networking.multicast airplay(play and pause button) not work
![IMG_4130](https://user-images.githubusercontent.com/12451477/215266056-58a67c64-0784-4521-a005-ab8a669f9a45.PNG)

**Comment by quanlv9695** [comment]
We got the same error,
Is it only iOS 16 devices?

**Comment by nicolasanteojos** [comment]
Same problem, can anyone solve it?

**Comment by nicolasanteojos** [comment]
> can anyone solve problem, please help me
We didn't move forward, but as far I understand this problem is iOS 16+, you need to add the entitlement:
"com.apple.developer.networking.multicast"
It have to be asked directly to Apple Support

**Comment by khanh0209ptit** [comment]
> ai có thể giải quyết được vấn đề xin hãy giúp tôi

Have you been able to fix this problem yet? Please advise

**Comment by sivaprasadkv** [comment]
Yes, this issue has been resolved by enabling Multicast Networking (com.apple.developer.networking.multicast).
Request Apple to enable this advanced capability in your developer account for a particular Apple identifier. 
Provide proper justification on using this capability in your app.

URL to make a request is: https://developer.apple.com/contact/request/networking-multicast

Ones Apple approved and enabled it, enable this capability to your bundle identifier in your developer account.
After this, add com.apple.developer.networking.multicastv = YES in your entitlement file in your project.

After all this, your app discovers all the devices immediately.
Happy Coding! :)
